### PR TITLE
Remove "~" character from counts [MAILPOET-3708]

### PR DESCRIPTION
**Pull request by lysyjan**
[MAILPOET-3708]

[MAILPOET-3708]: https://mailpoet.atlassian.net/browse/MAILPOET-3708

### CODE REVIEW

File: assets/js/src/newsletters/send/notification.jsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ let fields = [
       return segment.name;
     },
     getCount: function getCount(segment) {
-      return `~${parseInt(segment.subscribers, 10).toLocaleString()}`;
+      return `${parseInt(segment.subscribers, 10).toLocaleString()}`;
```
Comment (pavel-mailpoet) [inline assets/js/src/newsletters/send/notification.jsx:63]: Does this need to be in ``?

File: assets/js/src/newsletters/send/standard.jsx [hunk 2]
```diff
@@ -160,7 +160,7 @@ let fields = [
       return segment.name;
     },
     getCount: function getCount(segment) {
-      return `~${parseInt(segment.subscribers, 10).toLocaleString()}`;
+      return `${parseInt(segment.subscribers, 10).toLocaleString()}`;
```
Comment (pavel-mailpoet) [inline assets/js/src/newsletters/send/standard.jsx:163]: Does this need to be in ``?

File: assets/js/src/newsletters/types/welcome/scheduling.jsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ const segmentField = {
   placeholder: MailPoet.I18n.t('selectSegmentPlaceholder'),
   forceSelect2: true,
   values: availableSegments,
-  getCount: (segment) => `~${parseInt(segment.subscribers, 10).toLocaleString()}`,
+  getCount: (segment) => `${parseInt(segment.subscribers, 10).toLocaleString()}`,
```
Comment (pavel-mailpoet) [inline assets/js/src/newsletters/types/welcome/scheduling.jsx:30]: Does this need to be in ``?

File: assets/js/src/settings/components/segments_select.tsx [hunk 4]
```diff
@@ -16,7 +16,7 @@ export default (props: Props) => {
   const segments = useSelector(selector)().map((segment) => ({
     value: segment.id,
     label: segment.name,
-    count: `~${segment.subscribers}`,
+    count: `${segment.subscribers}`,
```
Comment (pavel-mailpoet) [inline assets/js/src/settings/components/segments_select.tsx:19]: Does this need to be in ``?

File: lib/Models/Subscriber.php [hunk 5]
```diff
@@ -168,7 +168,7 @@ public static function filters($data = []) {
       ->whereNull('deleted_at')
       ->count();
     $subscribersWithoutSegmentLabel = sprintf(
-      WPFunctions::get()->__('Subscribers without a list (~%s)', 'mailpoet'),
+      WPFunctions::get()->__('Subscribers without a list (%s)', 'mailpoet'),
```
Comment (pavel-mailpoet) [inline lib/Models/Subscriber.php:171]: Could we remove `WPFunctions::get()->__` and only use `__` here?

File: lib/Subscribers/SubscriberListingRepository.php [hunk 6]
```diff
@@ -233,7 +233,7 @@ public function getFilters(ListingDefinition $definition): array {
     $subscribersWithoutSegmentCount = $subscribersWithoutSegmentStats[$key];
 
     $subscribersWithoutSegmentLabel = sprintf(
-      WPFunctions::get()->__('Subscribers without a list (~%s)', 'mailpoet'),
+      WPFunctions::get()->__('Subscribers without a list (%s)', 'mailpoet'),
```
Comment (pavel-mailpoet) [inline lib/Subscribers/SubscriberListingRepository.php:236]: Could we remove WPFunctions::get()->__ and only use __ here?

**Review by pavel-mailpoet — APPROVED** [review]

**Review by veljkho — APPROVED** [review]

**Pull request merged**